packages/grid/src/scss/_core.scss

```diff
@@ -0,0 +1,40 @@
+@import 'variables';
+@import 'functions';
+@import 'mixins';
+
+/**
+ * Defines our initial grid context, including width, max-width, and padding
+ * constraints based off of the given breakpoint.
+ */
+.#{$prefix}--grid {
+  $margin: map-deep-get($breakpoints, sm, margin);
+
+  width: 100%;
+  max-width: map-deep-get($breakpoints, max, min-width);
+  padding-right: $margin;
+  padding-left: $margin;
+  margin-right: auto;
+  margin-left: auto;
+
+  @include breakpoint(md) {
```

Comment:
- [ ] I think it would be more future-safe if these padding rules were made from a for-loop reading the breakpoint variables

Reply:
Definitely down to add in a loop that adds in the breakpoint if the padding differs from the previous one 👍